cfme/tests/services/test_dynamicdd_dialogelement.py

```diff
@@ -381,17 +381,31 @@ def test_child_dialog_should_update_with_new_options_based_on_option_of_parent_d
     pass
 
 
-@pytest.mark.manual
-@pytest.mark.tier(3)
-def test_value_input_into_service_dialog_element():
+@pytest.mark.customer_scenario
+@pytest.mark.meta(automates=[1364407])
+@pytest.mark.tier(2)
+@pytest.mark.parametrize("import_data", [DatastoreImport("bz_1364407.zip", "bz_1364407", None)],
+                         ids=["datastore"])
+@pytest.mark.parametrize("file_name", ["bz_1364407.yml"], ids=["dynamic_dialog"])
+def test_update_dynamic_field_on_refresh(appliance, import_datastore, import_data, file_name,
```

Comment:
I don't see `file_name` used in the test case. Why is it needed as a parameter?

Reply:
yes, right we don't need that. Thanks  :)